src/stpipe/cmdline.py

```diff
@@ -120,7 +120,8 @@ def set_value(subconf, key, val):
             set_value(subconf.setdefault(root, {}), rest, val)
         else:
             val, comment = config._handle_value(val)
-            subconf[root] = FromCommandLine(val)
+            if val in ['input_file', 'output_file']:  # or isinstance(val, str)
```

Comment:
Not that I understand why `val` gets values such as 'input_file', but the `isinstance` just seems more general.

Reply:
ah, not `val`. Should have been `key`.

Reply:
My initial implementation was using `isinstance` but I thought that maybe we do not need to convert every parameter to `FromCommandLine` unless we know it is a file path.

If `isinstance` is preferable, I'll modify the code.

Reply:
`if key in ['input_file', 'output_file']` was too restrictive and resulted in numerous unit test failures in `jwst` due to incorrect path. Using `isinstance`, on the other hand, did not result in failures.